pkg/core/capability/manual/manual.go

```diff
@@ -0,0 +1,85 @@
+package manual
+
+import (
+	"errors"
+	"reflect"
+	"soarca/internal/logger"
+	"soarca/pkg/core/capability"
+	"soarca/pkg/interaction"
+	"soarca/pkg/models/cacao"
+	"soarca/pkg/models/execution"
+	"time"
+)
+
+var (
+	component = reflect.TypeOf(ManualCapability{}).PkgPath()
+	log       *logger.Log
+)
+
+const (
+	manualResultVariableName = "__soarca_manual_result__"
+	manualCapabilityName     = "soarca-manual-http"
+	fallbackTimeout          = time.Minute * 1
+)
+
+func New(controller interaction.ICapabilityInteraction,
+	channel chan interaction.InteractionResponse) ManualCapability {
+	// channel := make(chan interaction.InteractionResponse)
+	return ManualCapability{interaction: controller, channel: channel}
+}
+
+func init() {
+	log = logger.Logger(component, logger.Info, "", logger.Json)
+}
+
+type ManualCapability struct {
+	interaction interaction.ICapabilityInteraction
+	channel     chan interaction.InteractionResponse
+}
+
+func (manual *ManualCapability) GetType() string {
+	return manualCapabilityName
+}
+
+func (manual *ManualCapability) Execute(
+	metadata execution.Metadata,
+	context capability.Context) (cacao.Variables, error) {
```

Comment:
Context has all data we need to execute a command, right? Should we rename "context" to e.g. "commandContext"?  Just using "context" is at least to me confusing to interpret right away.